modules/metrics/README.md

```diff
@@ -25,7 +25,7 @@ A Meter metric, per status-code, tagged with:
 |-------------|----------------------------|------------------------------------------------------|
 | what        | "endpoint-request-rate"    | Enable/disable with ENDPOINT_REQUEST_RATE            |
 | status-code | *                          | "200", "404", "418", etc.                            |
-| unit        | "request"                  |                                                      |
+| unit        | "request/s"                | Requests per second.                                 |
```

Comment:
The `endpoint-request-rate` metric generated by internal Apollo in fact has the unit 'request', exactly like here. I think 'request/s' is a better unit, and slightly better than 'message/s'?